source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -834,15 +762,7 @@ def complete_transfer(pt_journey, transfer_path_pool, request, transfer_sections
     logger = logging.getLogger(__name__)
     logger.debug("completing walking transfer in pt journey")
 
-    direct_path_type = StreetNetworkPathType.DIRECT
-    real_mode = 'walking'
-
-    for s in transfer_sections:
-        fallback_extremity = PeriodExtremity(s.end_date_time, True)
-        transfer_journeys = transfer_path_pool.wait_and_get(
-            s.origin, s.destination, real_mode, fallback_extremity, direct_path_type, request
-        )
-
-        if transfer_journeys and transfer_journeys.journeys:
-            new_section = transfer_journeys.journeys[0].sections
-            s.street_network.CopyFrom(new_section[0].street_network)
+    for section in pt_journey.sections:
+        if section.type != response_pb2.TRANSFER:
+            continue
+        transfer_pool.wait_and_complete(section)
```

Comment:
How about this instead?
```suggestion
        if section.type == response_pb2.TRANSFER:
            transfer_pool.wait_and_complete(section)
```

Reply:
it's more like a personal choice :smile_cat: I did so coz I hate indentations in python...

Reply:
It's one more line and the indentation is there anyway :shrug: Let's stop doing Python if indentation is a problem :stuck_out_tongue: